pages/osx/pkgutil.md

```diff
@@ -10,6 +10,6 @@
 
 `pkgutil --check-signature {{filename.pkg}}`
 
-- List all the files for an installed package:
+- List all the files for an installed package (e.g. Microsoft Word):
```

Comment:
I'd rather wait for the opinions of others on this example. Seems good to me, though.

Reply:
It's ok to not explicitly mention the package in the command description and use a real value in the user token of the example command. We in fact do encourage explicit examples over more general tokens when possible. This seems as is (currently without the added parentheses).